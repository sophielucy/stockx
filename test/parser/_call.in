myvar;
studentAge;
print("hello");
myfunc(myvar);
get_string(my_int);
