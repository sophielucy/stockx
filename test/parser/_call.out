Id(myvar)
Id(studentAge)
Call(print, StringLiteral(hello))
Call(myfunc, Id(myvar))
Call(get_string, Id(my_int))
